agent/async/lru_cache_test.go

```diff
@@ -71,7 +95,7 @@ func TestLRUSizePurge(t *testing.T) {
 
 func TestLRUCacheConcurrency(t *testing.T) {
 	// prime a cache with %80 size
-	size := 100000
+	size := 10000
```

Comment:
Why did you change this?

Reply:
I think after adding the logic to remove existed from the list, the `Set` will takes longer. `100000` will cause timeout.

Reply:
Can you benchmark the performance impact here with `go bench`?  10000 is significantly less concurrency than 100000 and I'd like to make sure we're actually going to have a valuable test here.

Reply:
The concurrency is only 5, `100000` is the size of the list. Because it will iterate the whole list for every `Set` with this change, the bigger the size of the list, the longer it will take. And the benchmark test results  are:
```
Before the change:
size 100000: BenchmarkLRUCacheConcurrency-8   	2000000000	         0.08 ns/op
size 10000: BenchmarkLRUCacheConcurrency-8   	2000000000	         0.01 ns/op
After the change:
size 10000: BenchmarkLRUCacheConcurrency-8   	       2	 579316132 ns/op
size 10000: BenchmarkLRUCacheConcurrency-8   	2000000000	         0.01 ns/op
```